CinnVIIStarkMenu@NikoKrause/3.2/applet.js

```diff
@@ -3609,9 +3616,7 @@ MyApplet.prototype = {
 
         this.appsBox = new St.BoxLayout({ vertical: true });
 
-        this.searchBox = new St.BoxLayout({ style_class: 'menu-search-box' });
-        this.searchBox.add_style_class_name("starkmenu-search-box");
-        this.searchBox.set_style("padding-right: 0px;padding-left: 0px;height:26px;");
+        this.searchBox = new St.BoxLayout({ style_class: 'starkmenu-search-box' });
```

Comment:
Yes, the problem with that is that most themes add a left padding on there and it looks awkward in the stark menu.
I removed that class because the search bar doesn't need any shared styles from the menu, a priori.